sale_partner_company/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "sale partner companyy",
+    "summary": "Set sale company from partner",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/multi-company",
+    "author": "Camptocamp, BCIM, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "data": [
+        "views/partner_view.xml",
+    ],
+    "depends": [
+        "sale_stock",
+        "sale_stock_warehouse_multicompany",
```

Comment:
```suggestion
```

You don't need this dependency. You can use this module to set the right company on the SO and it will use the WH of the selling company. The 2 modules are independents

Reply:
Removed dependency